actions/wait-for-deployment/entrypoint.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+set -eu
+
+while true; do
```

Comment:
Would it make sense to limit the number of iterations here, like in the [add-runner-ip action](https://github.com/broadinstitute/datarepo-actions/blob/4affe76aa5ce28a601daef4769ffe39af9acd084/actions/add-runner-ip/entrypoint.sh#L13)?

Reply:
Similar to [the answer I left for Shelby](https://github.com/broadinstitute/datarepo-actions/pull/55#discussion_r590512172), I hesitate to add a retry loop. In particular, I wonder if it may take more than 5 retries for the deployment to return to normal.